dune

```diff
@@ -0,0 +1 @@
+(dirs :standard \ node_modules)
```

Comment:
We have to tell dune to include `node_modules` since it is excluded from the build directory as the cause of the current problem.

Reply:
Super, thanks!